ovs/extensions/storageserver/storagedriver.py

```diff
@@ -176,6 +176,9 @@ def load(service):
         :param service: Service for which the MDSClient needs to be loaded
         """
 
+        if service.storagerouter is None:
+            return None
```

Comment:
It's better to raise an error here. This only should happen when we made a logical error in our code so we should see that error fairly soon. If we return None, it might go unnoticed.